samples/api/hpp_hello_triangle/hpp_hello_triangle.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I don't think it's okay to replace the existing copyright. If you add to a file, please append your copyright instead.

Reply:
This is still open. This sample was done by Andreas (Nvidia).

Reply:
Fixed

Reply:
I had to revert fix, it caused CI checks for copyrights to fail